docs/content/features/auto-responses/index.md

```diff
@@ -81,6 +81,14 @@ substituted with the [captured text](/features/auto-responses/#capture-groups) t
 
 ![capture](./capture_demo.png)
 
+#### \[e script\]
+
+substituted with whatever is printed from the the starlark script that comes after the e.
+
+![eval](./eval_demo.png)
```

Comment:
I think maybe the first demo should be simpler (since regexes haven't even been introduced yet)